build_runner/lib/src/builder/post_process_builder.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:build/build.dart';
+
+/// A [PostProcessBuilder] runs in a special phase at the end of the build.
+///
+/// They are different from a normal [Builder] in several ways:
+///
+/// - They don't have to declare output extensions, and can output any file as
+///   long as it doesn't conflict with an existing one.
+/// - They can only read their primary input id.
```

Comment:
"They can only read their primary input". It isn't the ID that they are reading.

Reply:
done